src/lib/agent/LedgerService.ts

```diff
@@ -0,0 +1,181 @@
+import logger from '../logger';
+import { Wallet } from '../wallet/Wallet';
+
+export class LedgerService {
+  wallet: Wallet;
+  indy: Indy;
+  poolHandle?: PoolHandle;
+  authorAgreement?: AuthorAgreement;
+
+  constructor(wallet: Wallet, indy: Indy) {
+    this.wallet = wallet;
+    this.indy = indy;
+  }
+
+  async connect(poolName: string, poolConfig: PoolConfig) {
+    try {
+      logger.log(__dirname);
+
+      logger.log('Creating pool config');
```

Comment:
Maybe we can add some more details to log messages?  In this case we could add `poolName`.

Reply:
Updated